script/client/at/db/oracle.sql

```diff
@@ -0,0 +1,29 @@
+-- for AT mode you must to init this sql for you business database. the seata server not need it.
+CREATE TABLE undo_log
+(
+    id            NUMBER(19)    NOT NULL,
+    branch_id     NUMBER(19)    NOT NULL,
+    xid           VARCHAR2(100) NOT NULL,
+    context       VARCHAR2(128) NOT NULL,
+    rollback_info BLOB          NOT NULL,
+    log_status    NUMBER(10)    NOT NULL,
+    log_created   TIMESTAMP(0)  NOT NULL,
+    log_modified  TIMESTAMP(0)  NOT NULL,
+    PRIMARY KEY (id),
+    CONSTRAINT ux_undo_log UNIQUE (xid, branch_id)
+);
+
+COMMENT ON TABLE undo_log IS 'AT transaction mode undo table';
+
+-- Generate ID using sequence and trigger
+CREATE SEQUENCE undo_log_seq START WITH 1 INCREMENT BY 1;
+
+CREATE OR REPLACE TRIGGER undo_log_seq_tr
```

Comment:
code implement specify undo_log_seq.nextval. here need auto generate?

Reply:
I'm not sure, but MySQL and PG use auto_increment, does there has different in these?

Reply:
need `undo_log_seq`

Reply:
Had edited.